### PR TITLE
RepeatMasker and perl Text::Soundex recipes

### PR DESCRIPTION
**Pull request by pvanheus**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pvanheus** [comment]
Build fails because of rmblast dependency.

File: recipes/repeatmasker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package:
+    name: repeatmasker
+    version: 4.0.6
+source:
+    fn:  RepeatMasker-open-4-0-6.tar.gz # [linux]
```
Comment (bgruening) [inline recipes/repeatmasker/meta.yaml]: If you skip the osx builds you don't need to add linux to every line.

File: recipes/repeatmasker/RepeatMaskerConfig.pm [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  ## ===================
+  ## The path to Phil Green's cross_match program ( phrap program suite ).
+  ##   - Use cross_match version 980501 or later for best results
+  ##   - On a windows machine running the cygwin emulation software
+  ##     you might set this to something like this:
+  ##
+  ##            $CROSSMATCH_DIR = "/cygdrive/c/phrap";
+  ##            $CROSSMATCH_PRGM = "cross_match.exe";
+  ##
+    $CROSSMATCH_DIR = "/usr/local/bin";
```
Comment (bgruening) [inline recipes/repeatmasker/RepeatMaskerConfig.pm]: /usr/local/bin looks wrong

File: recipes/repeatmasker/RepeatMaskerConfig.pm [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    $RMBLASTN_PRGM = "$RMBLAST_DIR/rmblastn";
+    $RMBLASTX_PRGM = "$RMBLAST_DIR/blastx";
+    $RMBLASTDB_PRGM   = "$RMBLAST_DIR/makeblastdb";
+ 
+  ##
+  ## WUBLAST/ABBlast Location
+  ## ========================
+  ## Set the location of the WUBLAST/ABBlast programs and support utilities.
+  ##
+    $WUBLAST_DIR   = "/usr/local/abblast";
```
Comment (bgruening) [inline recipes/repeatmasker/RepeatMaskerConfig.pm]: Same here ...

**Comment by bgruening** [comment]
@pvanheus I merged in rmblast so this is in theory ready to go. I added a few things as review.

File: recipes/repeatmasker/RepeatMaskerConfig.pm (hunk 3 above)
Comment (pvanheus) [inline recipes/repeatmasker/RepeatMaskerConfig.pm]: Those paths are not used since rmblast is used as the search engine, not wublast or crossmatch.

**Comment by bgruening** [comment]
Thanks Peter!

**Pull request merged**